assessments/projects/emissary-ingress/self-assessment.md

```diff
@@ -0,0 +1,446 @@
+    - Unit Tests
+    - linting checks for code quality.
+    - Checks for Envoy protobuf changes to find impact on code.
+    - Validation of Envoy version.
+    - Integration Tests.
+    - Validation of Helm chart changes.
+    - Builds and updates container image and the registry.
+    - Container image scanning for vunerabilities(trivy)
+    - E2E tests are also run frequently.
+* Release process:
+  - emissary-ingress has well organised security release process with dedicated team that responds to vulnerabilites quickly as described in [Secure release process](https://github.com/emissary-ingress/emissary/blob/master/DevDocumentation/SECURITY_RELEASE_PROCESS.md)
+
+* Communication Channels. Reference where you document how to reach your team or
+  describe in corresponding section.
+  * Internal. How do team members communicate with each other?
+    Team members communicate with each other through the [Community Slack](https://a8r.io/slack), [Github issues](https://github.com/emissary-ingress/emissary/issues) or [Zoom meetings](https://ambassadorlabs.zoom.us/j/86139262248?pwd=bzZlcU96WjAxN2E1RFZFZXJXZ1FwQT09).
+  * Inbound. How do users or prospective users communicate with the team?
+    Users communicate with the team through the [Community Slack](https://a8r.io/slack), [Github issues](https://github.com/emissary-ingress/emissary/issues) or [Zoom troubleshooting meetings](https://us02web.zoom.us/j/83032365622).
+  * Outbound. How do you communicate with your users? (e.g. flibble-announce@
+    mailing list)
+    Team members communicate with users through the [Community Slack](https://a8r.io/slack).
+
+* Ecosystem
+ Emissary-ingress is a specialized control plane for Envoy Proxy. In this architecture, Emissary-ingress translates configuration (in the form of Kubernetes Custom Resources) to Envoy configuration. All actual traffic is directly handled by the high-performance Envoy Proxy. It can route traffic to kubernetes services and directly to the pods aswell as integrate with service meshes like consul, linkerd, isito.
+
+## Security issue resolution
+
+Related security flaws may be reported over the Slack channel or by email to the emissary ingress team. The group responds expeditiously.
+In addition, users may visit [status.getambassador](https://status.getambassador.io/) where any problems that arise while the active products (Emissary, Edge Stack, Telepresence, Ambassador Cloud) are being fixed will be reported.
+
+**Responsible Disclosures Process**
+
+Assisting with the vulnerability fix, the emissary ingress works with The Common Vulnerability Scoring System (also known as CVSS Scores). In Detail, As soon as the team is able to put up the fix, a patched version will be made available for serious severity problems .The issues with Medium CVSS score will be fixed in the next version, which usually happens in 4-6 weeks or less. In a later version, they address Low 
```

Comment:
This section is meant to show 
- Responsible disclosure - how and where an individual can report security vulnerabilities to the project. 
- Incident response - how does the project handle these reports incl. triaging, communicating to stakeholders, releasing the fix.

The content included seems to be the remediation communication. Please revisit to include relevant content. This is a good example for your reference - https://github.com/cncf/tag-security/blob/main/assessments/projects/pixie/self-assessment.md#security-issue-resolution. Kindly add references to the content if any.

Reply:
We have modified the incident response and responsible disclosure based on your suggestion.